configuration/white_labels/nc.py

```diff
@@ -594,7 +594,7 @@ def get_white_label(self) -> WhiteLabel:
                 "icon": {"_icon": "Disabled", "_classname": "option-card-icon"},
                 "text": {
                     "_label": "conditionOptions.disabled",
-                    "_default_message": "Have any disabilities that make you unable to work now or in the future",
+                    "_default_message": "Currently have any disabilities that make you unable to work now or in the future",
```

Comment:
@Linda-Vu just a flag that this will change for NC, I think it's minor but wanted to let you know.